report/renderer/topbar-features.js

```diff
@@ -273,20 +254,39 @@ export class TopbarFeatures {
     }
   }
 
-  _setupStickyHeaderElements() {
+  _setupStickyHeader() {
+    // Cache these elements to avoid qSA on each onscroll.
     this.topbarEl = this._dom.find('div.lh-topbar', this._document);
-    this.scoreScaleEl = this._dom.find('div.lh-scorescale', this._document);
-    this.stickyHeaderEl = this._dom.find('div.lh-sticky-header', this._document);
+    this.categoriesEl = this._dom.find('div.lh-categories', this._document);
+
+    // Only present in the DOM if it'll be used (>=2 categories)
+    try {
+      this.stickyHeaderEl = this._dom.find('div.lh-sticky-header', this._document);
+    } catch (e) {}
+
+    if (!this.stickyHeaderEl) return;
```

Comment:
Just curious, does sticking `return` in the catch block narrow the type correctly?

Reply:
good idea. done. (also moved it a bit)